### DISCUSSION TITLE
Vertical axis minimum value

### DISCUSSION THREAD

**Discussion by sam9090** [drivardxrm/Sparkline.PCF#9] [Q&A]
How do you set the vertical axis minimum value?

Consider you are plotting a bar chart with the following data points:
- 4000
- 4000
- 3900

The 3900 looks like it's 0. This is the same issue you get with Excel sparklines but Excel allows you to set the vertical axis minimum value to 0 to resolve the issue. See example here:
https://trumpexcel.com/sparklines/

**Comment by drivardxrm** [comment]
Hi @sam9090,
That's a very good point.
I see that there are min and max attributes that are available in the library I use to render the sparkline, I will include them in the next release and keep you updated.

Hope you like the control.
